src/components/User/Tags/Tags.Index.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <div class="tags">
+    <h2>{{ "tags-index.header" | globalize }}</h2>
```

Comment:
Use tags manager component instead of tags index

Reply:
fixed